### DISCUSSION TITLE
Keep disabled additional services in sync

### DISCUSSION THREAD

**Discussion by dahaupt** [ddev/ddev#5030] [Ideas]
**Context:**  
In our projects, we are using additional services defined with Docker Compose. They are disabled by default and added to Git (`.ddev/.disabled-services/docker-compose.*.yaml`). They will be enabled if needed with `ddev service enable <name>`. Therefore, we excluded `/.ddev/docker-compose.*.yaml` from Git.

**Problem:**  
In case the configuration of a disabled service is changed (in Git), the outdated service configuration still exists in `/.ddev/docker-compose.*.yaml`. This already led to an issue with difficult debugging. In this case, the developer needs to `disable/enable` the service again in order to get the updated service definition. 

**Proposal:**
Improve handling for a disabled service and always copy the config file when running `ddev start/restart` (of course only if the service is active).

**Comment by rfay** [comment]
This is a valuable discussion; I didn't really know that people were using that feature, which is not fully formed. 

In current HEAD (would love it if you could test) we have `ddev get --installed`, which shows versions, and `ddev get --remove`.

Would you please test those? It's pretty easy, https://github.com/ddev/ddev/releases or https://ddev.readthedocs.io/en/latest/developers/building-contributing/#testing-latest-commits-on-head

But I think you're asking for a feature that's not there, which is that `ddev service disable` should pay attention to more than the docker-compose.*.yaml
